docs/pipelines.md

```diff
@@ -1564,21 +1564,20 @@ no `runAfter` can be specified in `finally` tasks.
 
 ## Using Custom Tasks
 
+Custom Task has been migrated from `v1alpha1` to `v1beta1`, see the [migration doc](migrating-v1alpha1.Run-to-v1beta1.CustomRun.md) for details.
+
 **Note: This is only allowed if `enable-custom-tasks` is set to
 `"true"` in the `feature-flags` configmap, see [`install.md`](./install.md#customizing-the-pipelines-controller-behavior)**
 
 [Custom Tasks](https://github.com/tektoncd/community/blob/main/teps/0002-custom-tasks.md)
 can implement behavior that doesn't correspond directly to running a workload in a `Pod` on the cluster.
 For example, a custom task might execute some operation outside of the cluster and wait for its execution to complete.
 
-A PipelineRun starts a custom task by creating a [`Run`](https://github.com/tektoncd/pipeline/blob/main/docs/runs.md) instead of a `TaskRun`.
+A PipelineRun starts a custom task by creating a [`CustomRun`](https://github.com/tektoncd/pipeline/blob/main/docs/customruns.md) instead of a `TaskRun`.
```

Comment:
we need to rename runs.md to customruns.md for this link to work

Reply:
Ops, updated!